pom.xml

```diff
@@ -3,26 +3,98 @@
         <modelVersion>4.0.0</modelVersion>
         <groupId>pro.taskana</groupId>
         <artifactId>taskana-adapter-parent</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
+        <version>0.0.2-SNAPSHOT</version>
```

Comment:
We just released v0.1. We should use 0.1.1-SNAPSHOT now.

Reply:
good catch. but is already reverted in TSK-973